cmd/geth/accountcmd.go

```diff
@@ -303,6 +321,55 @@ func accountProofOfPossession(ctx *cli.Context) error {
 	return nil
 }
 
+func accountConfirmHardwareAddress(ctx *cli.Context) error {
+	if len(ctx.Args()) != 1 {
+		utils.Fatalf("Please specify the address to confirm on the hardware wallet.")
+	}
+
+	stack, _ := makeConfigNode(ctx)
+	am := stack.AccountManager()
+
+	address := common.HexToAddress(ctx.Args()[0])
```

Comment:
Before you asked for `len(ctx.Args()) != 1` so probably it means that the address is not in the first position, but then you are extracting the address from there... Or am I missing something?

Reply:
If it's different than 1 then it fails and exits :)

Reply:
yes, but that means that it has two or more params. My question is if the index `0` is still the address, or maybe `1`

Reply:
Yeah, the context starts from the arguments, it doesn't contain the program name or anything. You can see an example here: https://github.com/celo-org/celo-blockchain/blob/7a9fe442d0b8480a3e7a4f5a42ac288e0ba35cc8/cmd/geth/accountcmd.go#L257